staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors.go

```diff
@@ -23,6 +23,9 @@ import (
 
 // WebhookAccessor provides a common interface to both mutating and validating webhook types.
 type WebhookAccessor interface {
+	// GetUid gets a string that uniquely identifies the webhook.
+	GetUid() string
```

Comment:
Why is the name not enough?

Reply:
found @liggitt's comment above.

Reply:
some comment would help

Reply:
@liggitt pointed out two problems with the name in https://github.com/kubernetes/kubernetes/pull/78080#issuecomment-497203350:  (1) The webhook name is only scoped to the webhook configuration (it's considered fine for two different webhook configurations to have webhooks with the same name) and (2) we don't currently have any validation in place to prevent two webhooks within the same webhook configuration from having the same name (I've opened https://github.com/kubernetes/kubernetes/issues/78510 for this).

Reply:
I'll add a comment clarifying.